src/components/ShellView/ResultTable.vue

```diff
@@ -157,22 +177,28 @@ export default {
       }
       const tableFields = Object.keys(this.queryResult.rows[0]);
       const tableTypes = this.queryResult.dataTypes;
+
       tableFields.forEach((field) => {
         this.tableHeaders.push({
           text: field,
           type: tableTypes[field],
         });
       });
+
       const numRows = this.queryResult.rows.length;
       const start = (this.page - 1) * this.itemsPerPage;
       const end = Math.min(start + this.itemsPerPage, numRows);
-      const rowsForPage = this.queryResult.rows.slice(start, end);
+      let rowsForPage = this.queryResult.rows.slice(start, end);
+
       rowsForPage.forEach((row) => {
         this.rows.push([]);
         for (let key in row) {
           if (!row[key]) {
             this.rows[this.rows.length - 1].push('NULL');
           }
+          else if (row[key]._nodes && row[key]._rels) {
```

Comment:
This is not robust. You should use column type instead of relying on if it has `_nodes` and `_rels`.